packages/mobile/components/drawers/DrawerManager.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+    import { localize } from '@core/i18n'
+    import { Drawer } from '../../components'
```

Comment:
Feels strange to have an import like this from within the components folder :thinking: it's a strange one because it should be an atom since it's a generic component, but it's only mobile-specific...

Reply:
the drawer is very much a mobile only component, the import is terrible because we dont have path aliases working on mobile yet, i usually add a bit of redundancy in the import to find it easier in the future when we include the path aliases

Reply:
this reminds me, I'll remove the drawer we have in the shared folder